notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb

```diff
@@ -581,7 +581,7 @@
    "source": [
```

Comment:
<p>I think this is a really good example, and perhaps even one we should work through with a pen and paper! It isn't _so_ straightforward to read this chart -- try</p><p><br></p><p>```</p><p>chain = np.arange(500)</p><p>az.plot_autocorr(chain, max_lag=500);</p><p>```</p><p><br></p><p>what interpretation do we have that `n=182` has autocorrelation 0?</p>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/bayesian-model-evaluation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
(to be clear, i don't have a good answer for this right now)